server/pulp/server/webservices/views/util.py

```diff
@@ -1,15 +1,29 @@
 import functools
 import json
 import sys
+from datetime import datetime
 from functools import wraps
 
 from django.http import HttpResponse
 from django.utils.encoding import iri_to_uri
 
-from pulp.common import error_codes
+from pulp.common import dateutils, error_codes
 from pulp.common.util import decode_unicode, encode_unicode
+from pulp.server.compat import json_util
 from pulp.server.exceptions import PulpCodedValidationException, InputEncodingError
-from pulp.server.webservices.controllers.base import json_encoder as pulp_json_encoder
+
+
+def pulp_json_encoder(thing):
```

Comment:
@bmbouter, take a look at this:
https://github.com/ipanova/pulp/blob/730/server/pulp/server/webservices/views/util.py#L59

`generate_json_response_with_pulp_encoder` uses `pulp_json_encoder` to serialize the response.